lib/mpl_toolkits/axes_grid1/inset_locator.py

```diff
@@ -276,7 +456,7 @@ def inset_axes(parent_axes, width, height, loc=1,
                                        width, height,
                                        loc=loc,
                                        bbox_transform=bbox_transform,
-                                       **kwargs)
+                                       borderpad=borderpad)
```

Comment:
Just checking that this fully exhausts all kwargs that can be passed to `AnchoredSizeLocator`?